op-batcher/batcher/driver.go

```diff
@@ -606,3 +676,60 @@ func logFields(xs ...any) (fs []any) {
 	}
 	return fs
 }
+
+type TestBatchSubmitter struct {
+	bs  *BatchSubmitter
```

Comment:
```suggestion
	*BatchSubmitter
```
If we embed, all methods are automatically available and we don't need to forward e.g. `StartBatchSubmitting`. Or did you deliberately not embed?

Reply:
I had it embedded originally but it doesn't work because then you end up with copies of the variables whose changes aren't seen by the "main" driver, and it breaks several tests. For example calling "StopBatcher" on the TestDriver wasn't actually stopping the batcher.

Reply:
I see, but what about embedding a pointer?

Reply:
good point, done!